README.md

```diff
@@ -125,6 +125,59 @@ class SampleViewController: UIViewController {
 
 ```
 
+### BNPL
+
+```swift
+import UIKit
+import Credify
+
+class SampleViewController: UIViewController {
+
+    private let bnpl = serviceX.BNPL()
+    private var user: CredifyUserModel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let config = serviceXConfig(apiKey: API_KEY, env: .sandbox, appName: APP_NAME)
+        serviceX.configure(config)
+        
+        user = CredifyUserModel(id: "internal ID in your system", firstName: "Vũ", lastName: "Nguyển", email: "vu.nguyen@gmail.com", credifyId: nil, countryCode: "+84", phoneNumber: "0381239876")
+    }
+
+    /// This starts Credify SDK
+    /// You need to create "orderId" on your side.
+    func startBNPL(orderId: String) {
```

Comment:
@lqgioan How do you use `getBNPLAvailability` function?

Reply:
@ngo275 sorry for my bad, I updated it